File: market/test/integration/java/ru/yandex/market/logistics/utilizer/security/tvm/TvmSecurityTestWithDisabledTvm.java
package ru.yandex.market.logistics.utilizer.security.tvm;

import com.github.springtestdbunit.annotation.DatabaseSetup;
import org.junit.jupiter.api.Test;
import org.springframework.context.annotation.Import;

import ru.yandex.market.logistics.utilizer.base.AbstractContextualTest;
import ru.yandex.market.logistics.utilizer.config.SecurityTestConfig;

import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;

@Import(SecurityTestConfig.class)
public class TvmSecurityTestWithDisabledTvm extends AbstractContextualTest {

    @Test
    @DatabaseSetup(value = "classpath:fixtures/controller/support/finalize-cycle/before.xml")
    public void testRequestIsSuccessfulForUnsecuredHealthMethod() throws Exception {
        mockMvc.perform(put("/support/finalize-cycle/100500"))
                .andExpect(status().isOk());
    }
}

File: market/test/integration/kotlin/tvm/TvmSecurityTestWithDisabledTvm.kt
package ru.yandex.market.logistics.calendaring.tvm

import com.github.springtestdbunit.annotation.DatabaseSetup
import org.junit.jupiter.api.Test
import org.springframework.context.annotation.Import
import org.springframework.test.context.TestPropertySource
import org.springframework.test.web.servlet.request.MockMvcRequestBuilders
import org.springframework.test.web.servlet.result.MockMvcResultMatchers
import ru.yandex.market.logistics.calendaring.base.AbstractContextualTest
import ru.yandex.market.logistics.calendaring.config.SecurityTestConfig

class TvmSecurityTestWithDisabledTvm : AbstractContextualTest() {

    @Test
    @DatabaseSetup("classpath:fixtures/controller/booking/get-slot/slot-exists/before.xml")
    fun testRequestIsSuccessfulForUnsecuredHealthMethod() {
        mockMvc!!.perform(MockMvcRequestBuilders.get("/booking/1"))
            .andExpect(MockMvcResultMatchers.status().isOk)
    }
}
